### DISCUSSION TITLE
Is there a way to share the JIT cache between processes

### DISCUSSION THREAD

**Discussion by alonkukl** [jax-ml/jax#14195] [Q&A]
In my scenario, I have a couple of workers (processes) running on the same machine.
The workers run JAX and could benefit from sharing the JIT cache, is there a way to accomplish it?

**Comment by jakevdp** [comment]
I don't know of any way of sharing the in-memory cache between processes, but there is experimental support for on-disk peristent compilation cache It currently [only supports TPU](https://github.com/google/jax/blob/ef4c2a38d65f4111b54f08f0c2e065174dde9b40/jax/_src/dispatch.py#L1061), but a GPU implementation is currently in the works.
